src/tcp.c

```diff
@@ -166,17 +183,16 @@ enftun_tcp_connect(struct enftun_tcp* tcp,
 }
 
 int
-enftun_tcp_connect_any(struct enftun_tcp* tcp,
-                       int mark,
-                       const char** hosts,
-                       const char* port)
+enftun_tcp_native_connect_any(struct enftun_tcp_native* ctx,
+                              const char** hosts,
+                              const char* port)
 {
     int rc = 0;
     const char* host;
 
     for (host = *hosts; host != NULL; host = *++hosts)
     {
-        rc = enftun_tcp_connect(tcp, mark, host, port);
+        rc = enftun_tcp_native_connect(ctx, host, port);
```

Comment:
If `connect_any` stays generic (not just for native), then call `_connect` via the ops.
```
rc = tcp->ops.connect(tcp, host, port);
```